### PR TITLE
Ban use of `MessageChannel#do_send`

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: daemon/src/fan_out.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ where
     async fn handle(&mut self, message: M, _: &mut xtra::Context<Self>) {
         for receiver in &self.receivers {
             // Not sure why here is no `do_send_async` ...
```
Comment (klochowicz) [inline daemon/src/fan_out.rs]: nit: do we still need this comment?

**Review by klochowicz — APPROVED** [review]
LGTM! 👍

**Review by da-kami — APPROVED** [review]

**Comment by da-kami** [comment]
note: after having trouble receiving messages sent from taker to maker when running both (debug build production) on the `another-auto-rollover` I rebased on this branch to see if that helps. Will report back.

**Comment by da-kami** [comment]
> note: after having trouble receiving messages sent from taker to maker when running both (debug build production) on the `another-auto-rollover` I rebased on this branch to see if that helps. Will report back.

seems to be an unrelated issue, it appears I have a deadlock upon startup at the moment. Rocket is not starting up correctly.

**Review by da-kami — APPROVED** [review]

**Pull request merged**